test/ssltestlib.c

```diff
@@ -10,53 +10,21 @@
 #include <string.h>
 
 #include "internal/nelem.h"
+#include "internal/cryptlib.h" /* for ossl_sleep() */
```

Comment:
I'm surprised this actually works. I don't think `ossl_sleep` gets exported from libcrypto...so how are we able to use it in the test apps?

Reply:
It works because `ossl_sleep()` is declared inline :)

Reply:
And BTW it originally was inline in `test/ssltestlib.c`, too.

Reply:
Ah! Of course!